crates/engine/src/mock_rocksdb_engine.rs

```diff
@@ -206,20 +212,48 @@ impl SnapshotApi for RocksSnapshot {
     }
 }
 
-/// A transaction of the `RocksEngine`
-#[derive(Copy, Clone, Debug, Default)]
+/// Mock `RocksTransaction`
```

Comment:
Shall we rename this `RocksTransaction` as `MockRocksTransaction`? Is it better?

Reply:
It's meant to replace the normal `RocksTransaction` if the madsim compile flag is set, they should share the same name.